recipes/implot/all/CMakeLists.txt

```diff
@@ -1,20 +1,18 @@
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.15)
 project(implot LANGUAGES CXX)
 
 file(GLOB HEADER_FILES ${IMPLOT_SRC_DIR}/*.h)
 
 add_library(${PROJECT_NAME}
     ${IMPLOT_SRC_DIR}/implot.cpp
     ${IMPLOT_SRC_DIR}/implot_items.cpp
-    ${IMPLOT_SRC_DIR}/implot_demo.cpp
```

Comment:
We added this last year because someone requested it in https://github.com/conan-io/conan-center-index/pull/20374

Honestly I don't mind both options, but if it's useful? Don't know. Do you have any extra insight I might be missing? Thanks!

Reply:
Huh, that seems like a rather niche use case. None of the symbols in `implot_demo.cpp` are even declared in the public header, I think. I would have pushed back on the previous PR and directed them to convince the upstream project to refactor that functionality into the library itself, really. Nevertheless, it does not actively hurt either and I'm ok with keeping it intact. I'll simply include a comment this time to prevent any future head scratches.

Reply:
Reverted.